frontend/src/app/components/UpdateList.js

```diff
@@ -0,0 +1,75 @@
+import React from 'react';
+import classnames from 'classnames';
+import moment from 'moment';
+import LayoutWrapper from './LayoutWrapper';
+import { newsUpdateL10nId } from '../lib/utils';
+
+export function prettyDate(date) {
+  return moment(date).format('MMMM Do, YYYY');
```

Comment:
Depending on how picky we want to get here, we could steal some formatting stuff from [this date-formats module in Snooze Tabs](https://github.com/bwinton/SnoozeTabs/blob/master/src/lib/time-formats.js). It had a lot of back & forth with l10n folks to get it just right